Renewed/nl_nl.json

```
@@ -1700,32 +1700,32 @@
-	"gui.lotr.factions.unpledge.desc.2": "NEW >>> You will incur a large alignment penalty, and be unable to make another pledge for some time.",
-	"gui.lotr.factions.pledgeLocked": "NEW >>> You cannot earn above %1$s alignment because of your pledge to %2$s",
+	"gui.lotr.factions.pledge": "Beloof dienst",
+	"gui.lotr.factions.pledge.req": "(Heeft %s afstemming nodig)",
+	"gui.lotr.factions.pledge.desc.1": "Je diesnt beloven aan %s gaat je vriendschap geven en vele extra's, waaronder de toegankelijkheid naar gebieden veroveren.",
+	"gui.lotr.factions.pledge.desc.2": "Als je deze belofte maakt, mag je hem altijd breken - maar als je dit doet krijg je een grote straf.",
+	"gui.lotr.factions.pledged": "In plicht!",
+	"gui.lotr.factions.pledge.breakCooldown": "Je kan niet in dienst gaan bij %1$s, omdat de wereld je verbroken plicht aan %2$s nog niet vergeten is.",
+	"gui.lotr.factions.pledge.enemies": "Je kan niet in diesnt gaan bij %s omdat je goede relaties hebt met hun aartsvijanden. (%s)",
+	"gui.lotr.factions.pledge.enemies.1": "%s",
+	"gui.lotr.factions.pledge.enemies.2": "%s, %s",
+	"gui.lotr.factions.pledge.enemies.3": "%s, %s, %s",
+	"gui.lotr.factions.pledge.enemies.3+": "%s, %s, %s, en %d meer...",
+	"gui.lotr.factions.unpledge": "Belofte verbreken",
+	"gui.lotr.factions.unpledge.desc.1": "Weet je zeker dat je je plicht aan %s wilt verbreken?",
+	"gui.lotr.factions.unpledge.desc.2": "Je gaat een grote straf tegeomet, en je kan voor een tijd geen nieuwe plicht aannemen.",
+	"gui.lotr.factions.pledgeLocked": "Je kan niet boven %1$s aanhang krijgen omdat je in plicht bent bij %2$s",
 	"gui.lotr.factions.civilianKills.yes": "Deze factie verheerlijkt oorlogsmisdaden (het vermoorden van onschuldige vijandelijke burgers)",
 	"gui.lotr.factions.civilianKills.no": "Deze factie haat oorlogsmisdaden (het vermoorden van onschuldige vijandelijke burgers)",
-	"gui.lotr.message.dismiss": "NEW >>> Dismiss",
-	"gui.lotr.message.notDisplayedAgain": "NEW >>> This message will not be displayed again",
-	"gui.lotr.message.friendly_fire": "NEW >>> It looks like you're trying to attack an allied NPC!\n\nWith the Friendly Fire setting disabled, you are unable to attack allied NPCs, which stops you from accidentally harming your allies. To enable Friendly Fire, open the Middle-earth menu and navigate to Options.",
-	"gui.lotr.message.align_drain": "NEW >>> When you see this icon:\n\n\n\nyou are losing alignment with one or more factions due to holding positive alignment (+) with a Mortal Enemy of theirs.\nKeep an eye on your alignments!",
+	"gui.lotr.message.dismiss": "Afwijzen",
+	"gui.lotr.message.notDisplayedAgain": "Dit bericht wordt niet opnieuw getoond",
+	"gui.lotr.message.friendly_fire": "Het ziet er uit dat je een geallieerde NPC wilt aanvallen!\n\nMet de vriendelijke vuur selectie uit, kan je geen geallieerde NPCs slaan, ga naar de Middle-earth opties in het menu om dit aan te zetten",
```

```suggestion
	"gui.lotr.message.friendly_fire": "Het ziet er uit dat je een geallieerde NPC wilt aanvallen!\n\nMet de eigen vuur selectie uit, kan je geen geallieerde NPCs slaan, ga naar de Middle-earth opties in het menu om dit aan te zetten",
```

"Eigen vuur" is the official Dutch term. https://nl.wikipedia.org/wiki/Eigen_vuur one of the two alternatives could be used if you think that works better, but I think "eigen vuur" is fine.